miso-web/src/main/webapp/WEB-INF/pages/editWorkstation.jsp

```diff
@@ -0,0 +1,59 @@
+<%@ include file="../header.jsp" %>
+
+<div id="maincontent">
+  <div id="contentcolumn">
+    <h1>
+      <c:choose>
+        <c:when test="${miso:isAdmin()}">Edit</c:when>
+        <c:otherwise>View</c:otherwise>
+      </c:choose>
+      Workstation
+    </h1>
+
+    <c:if test="${miso:isAdmin()}">
+        <button id="save" type="button" class="fg-button ui-state-default ui-corner-all">Save</button>
+    </c:if>
+
+    <div class="breadcrumbs">
+      <ul>
+        <li>
+          <a href="<c:url value='/miso/'/>">Home</a>
+        </li>
+        <li>
+          <a href='<c:url value="/miso/workstation/list"/>'>Workstations</a>
+        </li>
+      </ul>
+    </div>
+    <br>
+
+
+    <form:form id="workstationForm" data-parslet-validate="" autocomplete="off" acceptCharset="utf-8">
+    </form:form>
+
+    <br>
+    <h1>Instruments</h1>
+    <div id="instrumentForm_testsError"></div>
+    <table id="instruments_table"></table>
+    <br>
+
+    <br>
+    <h1>Libraries</h1>
+    <div id="libraryForm_testsError"></div>
+    <table id="libraries_table"></table>
+    <br>
```

Comment:
Wrong form and list name on the error divs, but not they should be removed anyway since these collections are not updated from this page